src/main/scala/io/opentargets/etl/backend/spark/Helpers.scala

```diff
@@ -139,4 +176,54 @@ object Helpers extends LazyLogging {
 
     renameDataType(schema)
   }
+
+  /**
+    * Helper method to apply a function to an existing column so that from the point of view of the user the
+    * dataframe passed in is mutated.
+    * @param columnName to apply function to
+    * @param dataFrame to transform
+    * @param fun transformation to apply to `columnName` in `dataFrame`
+    * @return transformed dataframe
+    */
+  def applyFunToColumn(columnName: String,
```

Comment:
@JarrodBaker this function is a bit useless I am afraid. Spark DF are immutables and that makes things like using the same column for the transformation useful and practical making this cumbersome wrapping not very useful.
So you can make things like
```scala
df.withColumn("columnName", lower($"columnName"))
...
```
Although your function and the standard approach will face a wall when it is used for internal struct fields where another method needs to use where involves struct recreation [see this](https://stackoverflow.com/a/48790677)